### PR TITLE
RKMilCommute_Allocation

### PR DESCRIPTION
**Pull request by deeepeshthakur**
@ChrisRackauckas 
#169 #171

### CODE REVIEW

File: src/StochasticDiffEq.jl [hunk 1]
```diff
@@ -20,7 +20,7 @@ module StochasticDiffEq
 
   using Logging
 
-  using LinearAlgebra, Random
+  using LinearAlgebra, Random, FillArrays
```
Comment (ChrisRackauckas) [inline src/StochasticDiffEq.jl]: Need to add FillArrays.jl to the Project.toml file.

Comment (deeepeshthakur) [reply src/StochasticDiffEq.jl]: But Eye calls Diagonal. I not sure this will fix it.

Comment (ChrisRackauckas) [reply src/StochasticDiffEq.jl]: `Diagonal(n)` is not a base dispatch. It's a special FillArray thing that builds the representation without building the arrays of the matrix.

Comment (deeepeshthakur) [reply src/StochasticDiffEq.jl]: Oh. Nice!

**Pull request merged**